apps/mobile/src/app/_layout.tsx

```diff
@@ -35,9 +35,13 @@ configureReanimatedLogger({
   strict: false,
 })
 
-// Initialize store-side effects
-store.dispatch(apiSliceWithChainsConfig.endpoints.getChainsConfig.initiate())
-
+persistor.subscribe(() => {
+  const { bootstrapped } = persistor.getState()
+  if (bootstrapped) {
+    // The chain config is persisted in the store, but might be outdated.
+    store.dispatch(apiSliceWithChainsConfig.endpoints.getChainsConfig.initiate(undefined, { forceRefetch: true }))
+  }
+})
```

Comment:
Previously we were dispatching the fetch whenever the store was ready, now it's better to wait for the rehydrate action to complete and to dispatch after that.